pkg/util/system/system_utils.go

```diff
@@ -18,14 +18,25 @@ package system
 
 import (
 	"strings"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
I do not think it is right to import an application(`kubeadm`) code in a `util` package.